src/EnrollCallAndSmsController.js

```diff
@@ -274,8 +283,28 @@ function (Okta, FormController, Footer, PhoneTextBox, TextBox, CountryUtil, Form
         noButtonBar: true,
         autoSave: true,
         className: getClassName(factorType),
+
+        showWarning: function (msg) {
+          clearTimeout(warningTimeout);
+          this.clearWarnings();
+          this.add(warningTemplate, '.o-form-error-container', {options: {warning: msg}});
+        },
+        clearWarnings: function () {
+          this.$('.okta-form-infobox-warning').remove();
+        },
+
         initialize: function () {
+          this.listenTo(this.model, 'change:ableToResend', function (model, ableToResend) {
+            if (!ableToResend) {
+              warningTimeout = Util.callAfterTimeout(_.bind(function () {
```

Comment:
I don't think we need to create a timeout here... we probably can use the logic we already have for changing the button's text. Actually I don't think we need all any change to this initialize specifically. 
Take a look at the [initialize](https://github.com/okta/okta-signin-widget/pull/711/files#diff-12bdb1aff4a9e2a67ac933eb47538706R249) function for the retry button, that is called when `ableToResend` changes values. All that is already happening every 30 secs, that a look at [limitResending](https://github.com/okta/okta-signin-widget/pull/711/files#diff-12bdb1aff4a9e2a67ac933eb47538706R98) function

Reply:
fixed by putting it under initialize of Form